rest-service/manager_rest/storage/resource_models.py

```diff
@@ -967,22 +967,26 @@ def _system_workflow_task_name(self, wf_id):
             'delete_deployment_environment':
                 'cloudify_system_workflows.deployment_environment.delete',
             'update_plugin': 'cloudify_system_workflows.plugins.update',
-            'upload_blueprint': 'cloudify_system_workflows.blueprint.upload'
+            'upload_blueprint': 'cloudify_system_workflows.blueprint.upload',
+            'csys_update_deployment':
+            'cloudify_system_workflows.deployment_environment.'
```

Comment:
This would be nicer with a bit of indentation (similar to the delete_deployment_environment one.